contracts/evm/src/dvn/interfaces/ILayerZeroDVN.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Remove these copied interfaces please. They should be pulling from the dependency